matRadGUI.m

```diff
@@ -2451,7 +2451,8 @@ function getPlnFromGUI(handles)
 
 try
     cst = evalin('base','cst');
-    if sum(strcmp('TARGET',cst(:,3))) > 0 && get(handles.checkIsoCenter,'Value')
+    if (sum(strcmp('TARGET',cst(:,3))) > 0 && get(handles.checkIsoCenter,'Value')) || ...
+            (sum(strcmp('TARGET',cst(:,3))) > 0 && ~isfield(pln,'isoCenter'))
        pln.isoCenter = ones(pln.numOfBeams,1) * matRad_getIsoCenter(cst,ct);
```

Comment:
I am confused (as always with matRadGUI.m)....

~isfield(pln,'isoCenter')) is always true here because it has not been constructed yet as you build the pln struct from scratch in this sub function.